webview/src/shared/components/tooltip/styles.module.scss

```diff
@@ -30,3 +26,17 @@ $font: var(--vscode-font-family);
     z-index: 2;
   }
 }
+
+.tooltipContent {
+  border: 1px solid $tooltip-border;
+  white-space: pre-wrap;
+  padding: 5px;
```

Comment:
Should we decrease the plots info tooltips padding now that they already get some padding from `.tooltipContent`:

<img width="295" alt="image" src="https://user-images.githubusercontent.com/43496356/189129167-42454c29-eba6-40d3-b76b-1c127398c545.png">

Reply:
I changed the padding [here](https://github.com/iterative/vscode-dvc/pull/2353/files#diff-71037f545ae5c406885ca4620eaf6f314bcc77c43842d017d318cc958723c11cR368).